frontend/src/libraries/explorer-wamp/accounts.ts

```diff
@@ -197,6 +197,32 @@ export default class AccountsApi extends ExplorerApi {
     }
   }
 
+  async queryDbAccount(accountId: string): Promise<any> {
+    try {
+      let accountDetails;
+      if (this.dataSource === DATA_SOURCE_TYPE.INDEXER_BACKEND) {
```

Comment:
I have already created the `legacy` branch of near-explorer, so we can strip all these `dataSource` checks in the master branch.